Sources/Plasma/PubUtilLib/plAudio/plSound.cpp

```diff
@@ -757,34 +774,32 @@ bool plSound::ILoadDataBuffer()
 {
     if(!fDataBufferLoaded)
     {
-        plSoundBuffer *buffer = (plSoundBuffer *)fDataBufferKey->RefObject();
-        if(!buffer)
+        fDataBuffer = (plSoundBuffer*)fDataBufferKey->RefObject();
```

Comment:
I guess this was already a cast, but... ConvertNoRef? or actually, Convert (with ref) in this case?

Reply:
`ConvertNoRef` would be more appropriate here.